src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -151,9 +151,10 @@
         input-value    (reagent/atom "")
         input-focused? (reagent/atom false)]
     (fn []
-      (let [selected-tab          (or (rf/sub [:wallet/send-tab]) (:id (first tabs-data)))
-            valid-ens-or-address? (boolean (rf/sub [:wallet/valid-ens-or-address?]))
-            {:keys [color]}       (rf/sub [:wallet/current-viewing-account])]
+      (let [selected-tab             (or (rf/sub [:wallet/send-tab]) (:id (first tabs-data)))
+            valid-ens-or-address?    (boolean (rf/sub [:wallet/valid-ens-or-address?]))
+            local-suggestion-address (rf/sub [:wallet/local-suggestions->full-address])
```

Comment:
As an outsider to the wallet domain, I wondered why getting the first suggestion is important and I thought that reason could be on the name/keyword. In other words, why is the first suggestion relevant? What's the order? Why not the last? If that was in the name, it could help others understand. Also, the name is in the plural, which gives me the impression the sub will return a sequence of full addresses, and not just one.

Suggestions? `:wallet/top-suggestion-full-address`. Or if it's really arbitrary the choice of suggestion the name could be `:wallet/first-suggestion-full-address`.

Reply:
To be honest, I don't know even why it's returning an array. I thought it'd be useful later. At the moment, only one item is returned and even the implementation is for one ens match. And, each ens has only one address. So, only one item is stored in the db, but it's in an array.